packages/client/src/pages/Home/index.js

```diff
@@ -228,6 +235,39 @@ const WhyJoin = [
   }
 ]
 
+const Testimonials = [
+  {
+    key: 1,
+    text: 'home.testimonials.testimonial1', 
+    auth: 'home.testimonialauths.testimonial1auth'
+  },
+  {
+    key: 2,
+    text: 'home.testimonials.testimonial3',
+    auth: 'home.testimonialauths.testimonial3auth'
+  },
+  {
+    key: 3,
+    text: 'home.testimonials.testimonial5',
+    auth: 'home.testimonialauths.testimonial5auth'
+  },
+  {
+    key: 4,
+    text: 'home.testimonials.testimonial2',
+    auth: 'home.testimonialauths.testimonial2auth'
+  },
+  {
+    key: 5,
+    text: 'home.testimonials.testimonial4',
+    auth: 'home.testimonialauths.testimonial4auth'
+  },
+  {
+    key: 6,
+    text: 'home.testimonials.testimonial6',
+    auth: 'home.testimonialauths.testimonial6auth'
+  }
+]
+
```

Comment:
Making the testimonials an array would reduce the amount of hard code necessary.